src/main/java/com/mvelyka/library/Book.java

```diff
@@ -0,0 +1,16 @@
+package com.mvelyka.library;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
```

Comment:
We can user @Data instead all of: @Getter, @Setter, @ToString and @AllArgsConstructor